client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -370,6 +370,8 @@ class EmailProvidersComparison extends Component {
 
 		const { validatedMailboxUuids } = this.state;
 
+		const isEligibleForFreeTrial = hasCartDomain || isDomainEligibleForTitanFreeTrial( domain );
```

Comment:
This is obviously wrong since we're getting the eligibility for Titan instead of Google Workspace. I suspect that's on purpose as we don't have any selector for this yet but I think this could have been noted somewhere.

> Does this patch can go live before any other one?

Well, if you add this selector then I guess we can deploy this pull request safely even if the REST API has not been updated to return the eligibility status yet.

Reply:
Sorry, It was a draft and I removed the draft status, I didn't forgot I left that selector to be able to test / see the new notice.